CHANGES/6752.misc

```diff
@@ -0,0 +1 @@
+Added python3-psycopg2 to be installed for RH based distributions.
```

Comment:
We also need to update docs/index.md to state that users no longer need to install this.

And did you confirm that psycopg2 is not installed in the molecule container?
https://pulp.plan.io/issues/6752#note-11

Reply:
Any of created box doesn't have psyscopg package installed. 
Removed the line from the docs that user need self-install the package.

Reply:
Actually, this comment is now incorrect.